### DISCUSSION TITLE
Which version of the old wappalyzer code are we supposed to use ?

### DISCUSSION THREAD

**Discussion by abhijeetkushe** [enthec/webappanalyzer#110] [Q&A] [answered]
I dont see the source code for nodejs.So I assume this is technologies that are going to be added.Can you share what is the last known wappalyzer code that this supports or how are we supposed to install this package ?

**Comment by enthec-opensource** [comment, marked as answer]
This project aims to maintain just the technology detectors based on previous implementations made by the community and wappalyzer itself.
If you want to check the last copy of wappalyzer right before it went dark you can check it here: https://github.com/dochne/wappalyzer

There is no way to obtain the official wappalyzer project as it is private now, you would need to pay for their api or just sign up for a bunch of free get requests a month, more information here: https://www.wappalyzer.com/pricing/

This project keeps the same file structure as it was when the wappalyzer project was public, so every other community made library should work.

A simple github search gets me some js implementations that might work for you (not tested):
- https://github.com/Kikobeats/simple-wappalyzer
- https://github.com/TitovDigital/wappalyzer-js-cli

But any other outdated js implementation should work just by replacing `https://github.com/AliasIO/Wappalyzer/...` with `https://github.com/enthec/webappanalyzer/...`

**Comment by abhijeetkushe** [comment]
Thanks @enthec-opensource
